pages/blog/posts/manfred-case-study-es.md

```diff
@@ -21,7 +21,7 @@ excerpt: "Descubre cómo Manfred ha usado JSON Schema para transformar la indust
 
 Manfred inició su andadura con la ambiciosa misión de revolucionar el panorama de reclutamiento y para ello decidió apostar por la transparencia total y una estrategia Open Data. Las plataformas de reclutamiento tradicionales se han caracterizado por ofrecer un acceso limitado a los datos personales, a menudo proporcionando archivos poco accionables como PDFs. Manfred apostó por ofrecer una propuesta de valor única ofreciendo una solución que empoderara a los desarrolladores para poseer sus datos en el formato de intercambio universal: JSON.
 
-<div className="text-2xl my-5 mx-8 border-gray-300 bg-gray-300 p-4 p-t-6 text-center dark:bg-gray-900">
+<div className="text-2xl my-5 mx-8 border-gray-300 bg-gray-300 p-4 p-t-6 text-center dark:bg-gray-900 dark:text-slate-800">
```

Comment:
Why aren't we using the component here? Is this a specific use case?

Reply:
Good catch. Fixed